contracts/Hub.sol

```diff
@@ -0,0 +1,138 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.7;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+/**
+ * @title Hub Contract that coordinates Storage, Fund and Minter contract.
+ * @dev The contract can mint new tokens, check the collateral ratio of addresses
```

Comment:
instead of `new tokens` let us be more precise e.g : `new synthetic assets (erc20 token)`